ait/dsn/plugins/AOS_to_CCSDS.py

```diff
@@ -0,0 +1,85 @@
+from ait.core.server.plugins import Plugin
+from ait.core import log
+from ait.dsn.sle.frames import AOSTransFrame
+
+class AOS_to_CCSDS(Plugin):
+    '''
+    This plugin expects a stream of whole AOS frames, and outputs CCSDS packets
+    '''
+    def __init__(self, inputs=None, outputs=None, zmq_args=None, **kwargs):
+        super().__init__(inputs, outputs, zmq_args)
+        self.bytes_from_previous_frame = None
+
+    def process(self, data, topic=None):
+        AOS_frame_object = AOSTransFrame(data)
+        if AOS_frame_object.is_idle_frame:
+            log.debug(f"Dropping idle frame!")
+            return
+        else:
+            first_header_pointer = AOS_frame_object.get('mpdu_first_hdr_ptr')
```

Comment:
Should there be a check to ensure the AOS frame is M_PDU type before processing ?

Reply:
@kmarwah I added a check for M_PDU type into your commit.  Can you pull this branch and do a quick test with the fsws? It's down on my end.